### DISCUSSION TITLE
Support for references on all type of tokens

### DISCUSSION THREAD

**Discussion by Adsiezip** [six7/figma-tokens#660] [Q&A]
I have this issue where we have tokens for _space_. And then I wanna reference that token on another set by typing in the name in bracts like this {spacetokenname}. But then the plugin stops working. Not sure why?

**Comment by six7** [comment]
Sorry for the late reply, been busy with Config and the new release. Could you create an issue and add a recording? this should be possible!

**Reply by Adsiezip** [reply]
Thanks for the answer! I went to look again and I most have missed the little button on the right that said "Alias mode". Tested it and it worked. Thanks for the help! 
Love your plugin :D
